local stdio = require("@lune/stdio")
local b = stdio.style("bold") -- Yes I know the variable name is shortened, but in the context of this command, it's fine.
local re = stdio.style("reset")
local r = stdio.color("red")
return function()
    print(`usage: {r}rcc{re} [command] [options]`)
    print(`{b}Commands:{re}`)
    print(`  {b}help{re}\t\topen this help menu`)
    --print(`  {bold}init{reset}\t\tinitialize a project`)
    --print(`  {b}install{re}\tinstall a compiler`)
    --print(`  {b}uninstall{re}\tuninstall a compiler`)
    --print(`  {b}update{re}\tupdate a compiler`)
    print(`{b}Options:{re}`)
    --print(`  {b}-o{re}\t\toutput directory (default: out)`)
    --print(`  {b}<none>{re}\tinput directory (default: src)`)
    --print(`  {b}-d{re}\t\tdebug mode`)
    print(`{b}Examples:{re}`)
    --print(`  {b}rcc{re} {b}install{re} roblox-py`)
    --print(`  {b}rcc{re} {b}uninstall{re} roblox-py`)
    --print(`  {b}rcc{re} {b}update{re} roblox-py`)
    --print(`  {b}rcc{re} {b}-o{re} out`)
    --print(`  {b}rcc{re} {b}-o{re} out src`)
    --print(`  {b}rcc{re} {b}include{re} @roblox/roact`)
  end